packages/components/src/components/date-picker/date-picker.tsx

```diff
@@ -1,25 +1,46 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { addDays, daysFromTodayTo, toMoment, convertDateFormat, getPosition, isMobile } from '@deriv/shared';
 
-import Input from './date-picker-input.jsx';
-import Calendar from './date-picker-calendar.jsx';
-import Native from './date-picker-native.jsx';
+import Input from './date-picker-input';
+import Calendar from './date-picker-calendar';
+import Native from './date-picker-native';
 import MobileWrapper from '../mobile-wrapper';
 import DesktopWrapper from '../desktop-wrapper';
 import { useOnClickOutside } from '../../hooks/use-onclickoutside';
 
-const DatePicker = React.memo(props => {
+type TDatePickerOnChangeEvent = {
+    date?: string;
+    duration?: number | null | string;
+    target?: { name: string; value: number | string | moment.Moment | null };
+};
+export type TDatePicker = Omit<
```

Comment:
```suggestion
type TDatePicker = Omit<
```
@amir-deriv Avoid exporting component props type 🙏🏻
Ideally, the props type is only scoped to the component itself, So we shouldn't export or put the type in another file, And if we need the type we can use generics like `React.ComponentProps` to access it 🙇🏻

Reply:
Got it